src/components/donation/Steps/CurrentStep.tsx

```diff
@@ -23,6 +21,30 @@ export default function CurrentStep(props: ConfigParams) {
     !wallet && dispatch(resetDetails());
   }, [wallet, dispatch]);
 
+  // If there is no wallet connected take user the fiat-only path
+  if (!wallet) {
+    switch (state.step) {
+      case 3: {
+        return <FiatSubmit {...state} />;
```

Comment:
@SovereignAndrey, I tried to also remove duplicate markups like `<FiatSubmit/>` , `<FiatForm/>`
in 
* #2107 

the difference is `country` field is below `amount` when `token === "fiat"`

<img width="509" alt="image" src="https://user-images.githubusercontent.com/89639563/234466476-41417178-9239-4a3d-aed6-ba2141c05c8b.png">

removed from summary
![Uploading image.png…]()

Reply:
@ap-justin 
> the difference is country field is below amount when token === "fiat"
> removed from summary

What was the reason it could not stay on the summary page and use the payment type to toggle if it's visible or not? 

Judging by the screenshots above, seems like if a user switches to Fiat payment, then that field pops in and would disappear if they decided to choose a crypto payment? :thinking:

Reply:
> What was the reason it could not stay on the summary page and use the payment type to toggle if it's visible or not?

I noticed that summary page has been duplicated just to add `country` field
<img width="517" alt="image" src="https://user-images.githubusercontent.com/89639563/234479216-e64d740a-f651-47b2-84fb-926535d6571c.png">


also, putting this field below `token` makes it somehow connected to it 
like if `usd` provide `country`

<img width="505" alt="image" src="https://user-images.githubusercontent.com/89639563/234479025-7d8feca4-e9f2-4acd-ad0c-1b15f6be8896.png">

> Judging by the screenshots above, seems like if a user switches to Fiat payment, then that field pops in and would disappear if they decided to choose a crypto payment

Yup, if crypto payment (wallet is connected), the `tokens` to select is `cryptokens + fiats`
<img width="988" alt="image" src="https://user-images.githubusercontent.com/89639563/234479841-9a6dbe8b-89a6-433e-b03b-dae364c81dc3.png">

if fiat payment (no connected walelt), the `tokens` to select is `fiats` only
<img width="992" alt="image" src="https://user-images.githubusercontent.com/89639563/234479878-bae3bdb1-7944-4225-b0ca-9d763e532a52.png">

Reply:
@ap-justin Okay then. We'll go with that for now. Hopefully, product team is OK with those visual changes. :crossed_fingers:

Reply:
🤞 will open draft PR then

Reply:
🤞 will open draft PR then

Reply:
🤞 will open draft PR then